src/audit/mod.rs

```diff
@@ -0,0 +1,25 @@
+use std::{env, fs};
+use uuid::Uuid;
+
+/// Internal logging function used to log errors.
+/// Can be disabled by setting environment variable ACCEPTEVM to 0
+pub fn log_sync(data: &str) {
```

Comment:
the name of module is not clear. in `audit` i would like to see some audit reports

Reply:
e889da77a756abf1d3390a98806640e31f58cfb7